README.md

```diff
@@ -14,7 +14,6 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
```

Comment:
- [Optional] Please kindly rephrase the first line of the project description. It is not clear enough. Also, the word "Microverse" was not correctly spelled and don't forget that it is a proper noun. Let's kindly capitalize the name :+1:  

 > This project is math-magician-projects microvers which will be used for learning in the microverse. No complex codind is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!. 1- react 2- css 3- linters 4- javascript 5-modules

Kindly  read through the entired paragraph, rephrase your lines, correct typo. This would render your README file more clearer.